### ISSUE TITLE
Mapping Tools became unusable.

### ISSUE THREAD

**Issue by AARaidenMK** [OliBomby/Mapping_Tools#206]
## Prerequisites
- [x] I have read the [Mapping Tools FAQ](https://mappingtools.github.io/faq)

## Environment

**OS version**: Windows 11

**Mapping Tools version**: 1.12.17

## Description
I just ran into this bug today and it's unbearable because I can't figure out how to fix it at all. Let me explain...

When I opened Mapping Tools, I wanted to fetch any difficulty from osu! client so I can add hitsounds to one of them. But, every time I try fetching a difficulty, I get an error message. This also happens when you try to open current beatmap.

After reinstalling Mapping Tools, it didn't solve the issue. I did save my pattern gallery before I deleted Mapping Tools, and when I try to add an object to the gallery to go to the 'Pattern Files' the fast way, I get another error message saying 'Access denied'.

### Screenshots/Recordings
![Screenshot 2024-07-09 150521](https://github.com/OliBomby/Mapping_Tools/assets/54654438/9d0ce882-b1b8-4893-898e-92c9d19b5731)
![Screenshot 2024-07-09 150531](https://github.com/OliBomby/Mapping_Tools/assets/54654438/c251bc18-aff8-48a6-b55f-4f506660c45d)

## Additional context

**Comment by OliBomby** [comment]
This looks like an issue with Editor Reader. Something could be blocking access to reading the memory of osu! Maybe its an antivirus, firewall, or lacking permissions.

Until its fixed, i recommend disabling Editor Reader in the Preferences to prevent errors and not use "open current beatmap". Mapping Tools will at least be usable.

**Comment by TheFantasticLoki** [comment]
Hey just ran into this but it was because I run osu as admin for better frametimes so Hey if you run into this check that as well XD.

**Comment by QnoraeT** [comment]
I'm having similar problems with osu!lazer, and it is making Mapping Tools nearly useless, as I am unable to select any beatmaps (or, if there is a way, it has eluded me so far).

**Comment by OliBomby** [comment]
> I'm having similar problems with osu!lazer, and it is making Mapping Tools nearly useless, as I am unable to select any beatmaps (or, if there is a way, it has eluded me so far).

To use Mapping Tools for lazer refer to this: https://mappingtools.github.io/docs/guides/use-mt-in-lazer

Tldr not every feature of Mapping Tools is supported in lazer.
